---
abstract: The rapid development of second-generation sequencing has brought about
  a significant increase in the amount of omics data. Integrating and analyzing these
  single-cell datasets is a challenging problem. In this paper, we propose a new model,
  called as CVQVAE, based on a cross-trained VAE, and strengthened by the Vector Quantization
  technique for multi-omics data integration. CVQVAE projects data vectors from different
  omics onto a common latent space in such a way that (1) similar cells are close
  in the latent space and (2) the original biological information present in each
  of the omics (including cell cycle and trajectory) are preserved. Our model is trained
  and optimized solely based on the multi-omics data and requires no additional information
  such as cell-type labels. We empirically demonstrate the stability and efficiency
  of our method in data integration (alignment) on datasets from a recent competition
  on Open Problems in Single Cell Analysis.
title: 'CVQVAE: A representation learning based method for multi-omics single cell
  data integration'
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: liu22a
month: 0
tex_title: 'CVQVAE: A representation learning based method for multi-omics single
  cell data integration'
firstpage: 1
lastpage: 15
page: 1-15
order: 1
cycles: false
bibtex_author: Liu, Tianyu and Greenberg, Grant and Shomorony, Ilan
author:
- given: Tianyu
  family: Liu
- given: Grant
  family: Greenberg
- given: Ilan
  family: Shomorony
date: 2022-12-19
address:
container-title: Proceedings of the 17th Machine Learning in Computational Biology
  meeting
volume: '200'
genre: inproceedings
issued:
  date-parts:
  - 2022
  - 12
  - 19
pdf: https://proceedings.mlr.press/v200/liu22a/liu22a.pdf
extras: []
# Format based on citeproc: http://blog.martinfenner.org/2013/07/30/citeproc-yaml-for-bibliographies/
---
